terraform/setup/main.tf

```diff
@@ -31,20 +31,34 @@ resource "aws_security_group" "jenkins" {
     cidr_blocks = [var.ingress_cidr]
   }
 
+  ingress {
+    from_port   = 443
+    to_port     = 443
+    protocol    = "tcp"
+    cidr_blocks = [var.ingress_cidr]
+  }
+
   ingress {
     from_port   = 8080
     to_port     = 8080
     protocol    = "tcp"
     cidr_blocks = [var.ingress_cidr]
   }
+
+  egress {
+    from_port   = 0
+    to_port     = 0
+    protocol    = "-1"
+    cidr_blocks = ["0.0.0.0/0"]
+  }
```

Comment:
What is protocol "-1" ?

Reply:
@Cabalist that specifies all protocols (https://stackoverflow.com/questions/45519815/what-does-protocol-1-on-aws-security-group-egress-mean). This recreates the default aws "ALLOW ALL" egress rule. I got this block from the terraform documentation here: https://www.terraform.io/docs/providers/aws/r/security_group.html